classes/ObjectModel.php

```diff
@@ -1767,15 +1767,15 @@ public function deleteImage($force_delete = false)
                 && !unlink(_PS_TMP_IMG_DIR_.$this->def['table'].'_mini_'.$this->id.'.'.$this->image_format)) {
                 return false;
             }
-            if($this->isMultishop()){
+            if ($this->isMultishop()){
 
                 $ids_shop = $this->id_shop;
```

Comment:
Could be better with camelcase instead of underscore variable :)

Reply:
sure